src/main/java/org/apereo/portlet/soffit/connector/PortletApiSoffitLoader.java

```diff
@@ -49,6 +46,8 @@
 @Component
 public class PortletApiSoffitLoader extends AbstractSoffitLoader {
 
+    public static final String WINDOW_ID_PREFIX = "w_";
```

Comment:
Used to make the windowId member begin with a letter, and thereby more usable in Javascript, etc.